packages/docs/guide/index.md

```diff
@@ -36,9 +36,4 @@ See the list of config options in the [Config Reference](../config/app-configs.m
 
 ## Command Line Interface
 
-In a project where Figus is installed, you can use the `figus` binary in your npm scripts, or run it directly with `npx @figus/cli`.
-
-To download and generate icons, just run: `figus`
-
-
-## Examples
+To download and generate icons, just run: `figus`
```

Comment:
This arguably shouldn't be here, because they haven't setup their configuration yet and this will fail.